bin/setup

```diff
@@ -7,6 +7,9 @@ include FileUtils
 ENV["RAILS_ENV"]               ||= ENV["PASSENGER_APP_ENV"] || "development"
 ENV["RAILS_RELATIVE_URL_ROOT"] ||= ENV["PASSENGER_BASE_URI"]
 
+# if relative url root not set, use default for this app
+ENV['RAILS_RELATIVE_URL_ROOT'] ||= "/pun/sys/dashboard" if ENV['RAILS_ENV'] == 'production'
+
```

Comment:
Consider removing all usages of `OOD_PORTAL` and `OOD_SITE` in this file as they are unnecessary now.

Reply:
04c7482